src/vm/database/sqlite.rs

```diff
@@ -43,9 +48,17 @@ impl ContractDatabaseConnection {
                        value_type TEXT NOT NULL,
                        UNIQUE(contract_name, map_name))",
                             NO_PARAMS);
+        contract_db.execute("CREATE TABLE IF NOT EXISTS variables_table
+                      (variable_identifier INTEGER PRIMARY KEY AUTOINCREMENT,
+                       contract_name TEXT NOT NULL,
+                       variable_name TEXT NOT NULL,
+                       value_type TEXT NOT NULL,
+                       UNIQUE(contract_name, variable_name))",
+                            NO_PARAMS);
         contract_db.execute("CREATE TABLE IF NOT EXISTS data_table
                       (data_identifier INTEGER PRIMARY KEY AUTOINCREMENT,
                        map_identifier INTEGER NOT NULL,
+                       variable_identifier INTEGER NOT NULL,
```

Comment:
Why are there distinct `map_identifier` and `variable_identifier` fields?  Is it possible that both of these fields can be set?  Is it possible that neither of these fields can be set?  Should we consolidate these two fields into a `data_type` field whose value indicates that the row is a map, a variable, etc.?